diff --git a/Makefile.am b/Makefile.am
index 7b1b1bc..9d8d376 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2,9 +2,9 @@ if BUILD_WCAP_TOOLS
 wcap_subdir = wcap
 endif
 
-SUBDIRS = shared src clients data protocol tests $(wcap_subdir) man
+SUBDIRS = shared src clients data protocol $(wcap_subdir) man
 
-DIST_SUBDIRS = shared src clients data protocol tests wcap man
+DIST_SUBDIRS = shared src clients data protocol wcap man
 
 
 DISTCHECK_CONFIGURE_FLAGS = --disable-setuid-install
