diff --git a/clients/window.c b/clients/window.c
index 9bc9e19..fb1e164 100644
--- a/clients/window.c
+++ b/clients/window.c
@@ -933,10 +933,10 @@ window_create_surface(struct window *window)
 {
 	cairo_surface_t *surface;
 	uint32_t flags = 0;
-	
+
 	if (!window->transparent)
 		flags = SURFACE_OPAQUE;
-	
+
 	switch (window->buffer_type) {
 #ifdef HAVE_CAIRO_EGL
 	case WINDOW_BUFFER_TYPE_EGL_WINDOW:
@@ -3351,7 +3351,7 @@ menu_button_handler(struct widget *widget,
 	    time - menu->time > 500) {
 		/* Either relase after press-drag-release or
 		 * click-motion-click. */
-		menu->func(menu->window->parent, 
+		menu->func(menu->window->parent,
 			   menu->current, menu->window->parent->user_data);
 		input_ungrab(input);
 		menu_destroy(menu);
@@ -3749,7 +3749,7 @@ init_egl(struct display *d)
 #endif
 
 	static const EGLint argb_cfg_attribs[] = {
-		EGL_SURFACE_TYPE, EGL_WINDOW_BIT | EGL_PIXMAP_BIT,
+		EGL_SURFACE_TYPE, EGL_WINDOW_BIT,
 		EGL_RED_SIZE, 1,
 		EGL_GREEN_SIZE, 1,
 		EGL_BLUE_SIZE, 1,
@@ -4081,7 +4081,7 @@ display_release_window_surface(struct display *display,
 {
 #ifdef HAVE_CAIRO_EGL
 	cairo_device_t *device;
-	
+
 	device = cairo_surface_get_device(window->cairo_surface);
 	if (!device)
 		return;
