diff --git a/clients/Makefile.am b/clients/Makefile.am
index 99a5e3f..52b1b38 100644
--- a/clients/Makefile.am
+++ b/clients/Makefile.am
@@ -1,12 +1,12 @@
 bin_PROGRAMS =					\
 	weston-info				\
-	$(terminal)
-
-noinst_PROGRAMS =				\
 	$(clients_programs)			\
-	$(poppler_programs)			\
 	$(simple_clients_programs)		\
 	$(simple_egl_clients_programs)
+	$(terminal)
+
+noinst_PROGRAMS =				\
+	$(poppler_programs)
 
 libexec_PROGRAMS =				\
 	$(desktop_shell)			\
