diff --git a/configure.ac b/configure.ac
index 4193496..cd1b9f4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1636,9 +1636,9 @@ if test "x$enable_gallium_llvm" = xauto; then
 fi
 if test "x$enable_gallium_llvm" = xyes; then
     if test "x$llvm_prefix" != x; then
-        AC_PATH_PROG([LLVM_CONFIG], [llvm-config], [no], ["$llvm_prefix/bin"])
+        AC_PATH_PROG([LLVM_CONFIG], [llvm-config-2.9], [no], ["$llvm_prefix/bin"])
     else
-        AC_PATH_PROG([LLVM_CONFIG], [llvm-config], [no])
+        AC_PATH_PROG([LLVM_CONFIG], [llvm-config-2.9], [no])
     fi
 
     if test "x$LLVM_CONFIG" != xno; then
